src/components/Dashboard/Order/__tests__/OrdersSearch.test.js

```diff
@@ -0,0 +1,27 @@
+import OrdersSearch from '../OrdersSearch';
+
+const defaultProps = {
+  handleSelect: jest.fn(),
+  handleChange: jest.fn(),
+  title: 'Search',
+};
+
+const setup = buildSetup(OrdersSearch, defaultProps);
+
+describe('<OrdersSearch>', () => {
+  it('should render without crash', () => {
+    const { wrapper } = setup();
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should call handleSelect if value is selected', () => {
+    const { wrapper } = setup();
+    wrapper.find('select').simulate('change', { target: { value: 'realized' } });
+    expect(wrapper.instance().props.handleSelect).toHaveBeenCalled();
+  });
+  it('should call handleChange if if input value change', () => {
```

Comment:
(minor)
`if if`
also, there are inconsistencies in code formatting in this file. Between test case are empty lines, or they are missing.
It affects readability of the tests